src/Command/Project/ProjectListCommand.php

```diff
@@ -40,44 +40,38 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $refresh = $input->hasOption('refresh') && $input->getOption('refresh');
 
         // Fetch the list of projects.
-        $progress = new ProgressMessage($output);
-        $progress->showIfOutputDecorated('Loading projects...');
-        $projects = $this->api()->getProjects($refresh ? true : null);
-        $progress->done();
+        $subscriptions = $this->api()->getSubscriptions($refresh ? true : null);
```

Comment:
This is only fetching projects the user owns. The command needs to display the projects the user can access.